src/script/components/MessagesList/Message/ContentMessage/ContentMessage.tsx

```diff
@@ -163,134 +166,138 @@ export const ContentMessageComponent = ({
   useClickOutside(messageRef, hideActionMenuVisibility);
 
   return (
-    <div
-      aria-label={messageAriaLabel}
-      className="content-message-wrapper"
-      ref={contentMessageWrapperRef}
-      onMouseEnter={() => {
-        // open another floating action menu if none already open
-        if (!isMenuOpen) {
-          setActionMenuVisibility(true);
-        }
-      }}
-      onMouseLeave={() => {
-        // close floating message actions when no active menu is open like context menu/emoji picker
-        if (!isMenuOpen) {
-          setActionMenuVisibility(false);
-        }
-      }}
-    >
-      {(was_edited || !hideHeader) && (
-        <MessageHeader onClickAvatar={onClickAvatar} message={message} focusTabIndex={messageFocusedTabIndex}>
-          {was_edited && (
-            <span className="message-header-label-icon icon-edit" title={message.displayEditedTimestamp()}></span>
-          )}
-
-          <span className="content-message-timestamp">
-            <MessageTime timestamp={timestamp} data-timestamp-type="normal">
-              {timeAgo}
-            </MessageTime>
-          </span>
-        </MessageHeader>
-      )}
-
+    <>
       <div
-        className={cx('message-body', {
-          'message-asset': isAssetMessage,
-          'message-quoted': !!quote,
-          'ephemeral-asset-expired': isObfuscated && isAssetMessage,
-          'icon-file': isObfuscated && isFileMessage,
-          'icon-movie': isObfuscated && isVideoMessage,
-        })}
-        {...(ephemeralCaption && {title: ephemeralCaption})}
+        aria-label={messageAriaLabel}
+        className="content-message-wrapper"
+        css={{
+          width: `calc(100% - ${deliveryIndicatorRef?.offsetWidth || 0}px)`,
```

Comment:
I tried playing with `display: flex` and `overflow` etc. and with some of the message content (images, `pre`) the message was stretching outside of the screen